python/tirex_tracker/__init__.py

```diff
@@ -9,6 +9,7 @@
 from io import BytesIO
 from json import dumps, loads
 from pathlib import Path
+import shutil
```

Comment:
Can we use qualified imports in the main library code?
E.g., use `from shutil import copy` and later use just `copy` instead of `shutil.copy`.